pandas/sparse/frame.py

```diff
@@ -25,9 +25,14 @@
                                    create_block_manager_from_arrays)
 import pandas.core.generic as generic
 from pandas.sparse.series import SparseSeries, SparseArray
+from pandas._sparse import BlockIndex, get_blocks
 from pandas.util.decorators import Appender
 import pandas.core.ops as ops
 
+try:
```

Comment:
so we don't want to do an import here, which will affect all of pandas. 

- create a function ``pandas.types.common.is_scipy_sparse``

then can either: 

- do the import (and thus check ``isinstance``), you *may* need to jump some hoops to avoid repeatedly trying this import (you make a ``_try_import_scipy_sparse`` function), then assign the results to a module level variable on first use. see in ``pandas.io.pytables`` how ``_tables()`` works.

- OR, and maybe better, if scipy sparse matrixes are duck-type inferable, IOW, if its possible to check if they have a couple of key attributes then I would just do this.